group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroupTest.java

```diff
@@ -564,9 +564,9 @@ public void testMetadataRefreshDeadline() {
         assertTrue(group.hasMetadataExpired(time.milliseconds()));
 
         // Set the refresh time deadline with a higher group epoch.
-        group.setMetadataRefreshDeadline(time.milliseconds() + 1000, group.groupEpoch() + 1);
```

Comment:
why did we remove +1000? Wouldn't a deadline set to the current time be expired anyway (regardless of epoch)

Reply:
ah right, i did it wrong here. let me revert.

Reply:
fixed.